### ISSUE TITLE
[Bug]: narrator does not announce a dropdown option that is in hebrew when other options are in English

### ISSUE THREAD

**Issue by RanaKh1** [microsoft/fluentui#23168]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Window 11, Browser- Edge
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codepen.io/rana-khell/pen/NWyavQZ

## Actual Behavior
When focus lands on first option, narrator only announces one of seven, it does not announce the first option "תפוח"

## Expected Behavior
When focus lands on first option, narrator should announce: "one of seven, תפוח"

If the lang attribute is added to the button component used for the option, the narrator will announce the first option properly. I tried that by adding the attribute manually to the DOM
![image](https://user-images.githubusercontent.com/86226959/170008912-17b49def-de5f-4634-bfe6-b48a66e335af.png)

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by gouttierre** [comment]
@RanaKh1 - Thanks for filing this issue with us. I have added this to our accessibility queue for further investigations. To set expectations, the developers will review this issue once capacity allows.
